isis/src/control/objs/BundleSettings/BundleSettings.cpp

```diff
@@ -336,6 +343,27 @@ namespace Isis {
     }
   }
 
+  /**
+   * Set the normalized residual outlier rejection options for the bundle adjustment.
+   *
+   * @param grossOutlierRejection Indicates whether to perform automatic outlier
+   *                              rejection based on the normalized residuals
+   *                              during the bunlde adjustment.
+   *
+   * @param probabilityLevel The probability level the normalized residual 
+   *                         must fit within to be considered an inlier.
+   */
+  void BundleSettings::setGrossOutlierRejection(bool grossOutlierRejection, 
+						double probabilityLevel) {
+    m_grossOutlierRejection = grossOutlierRejection;
+    if (m_grossOutlierRejection) {
+      m_grossOutlierProbabilityLevel = probabilityLevel;
+    }
+    else {
+      m_grossOutlierProbabilityLevel = 0.95;
+    }
+  }
```

Comment:
I do not like this paradigm even though it matches the other outlier rejection setter. Instead I would always set the grossOutlierProbabilityLevel, the default argument will allow users to go `settings.setGrossOutlierRejection(false)` and get the desired result.

Reply:
I am a bit confused as to what you mean, because the m_grossOutlierProbabilityLevel is always being set. Are you saying we should only set m_grossOutlierProbabilityLevel if m_grossOutlierRejection is true? If so, then I agree!

Reply:
I would just set both values with what's passed in regardless. If grossOutlierRejection is false, then it doesn't matter what probabilityLevel is set to and it makes more sense to me to use what the user passed in. Only using the second value passed in if the first value is True is odd behavior for a setter, especially if the second value is optional. If the caller wanted to it set a specific value we should try to honor that. Here's what I'm thinking:

```
  void BundleSettings::setGrossOutlierRejection(bool grossOutlierRejection, 
						double probabilityLevel) {
    m_grossOutlierRejection = grossOutlierRejection;
    m_grossOutlierProbabilityLevel = probabilityLevel;
  }
```